docs/contributing/development-guide.md

```diff
@@ -231,92 +255,97 @@ docstrings must be formatted manually. To play nicely with Jupyter and IPython,
+structure and content of responses.
+
+The mock API object checks if `swagger.json` is accessible online; if not,
+it uses a local version located in the `tests/resources` directory within
+`stravalib`.
+:::
 
-1. functional end-to-end test suite: this test set requires an API key to run.
-1. unit tests that are setup to run on CI. These tests use mock
-   instances of the API to avoid needed to setup an API key yourself locally.
 
-### Unit - and integration test suite
+### Unit and integration test suite
 
-```{warning}
-We will add more information about the test suite in the near future.
-
-For integration tests that should be run independently from Strava, there's a
-pytest fixture {py:func}`~stravalib.tests.integration.conftest.mock_strava_api`
-that is based on {py:class:}`responses.RequestsMock`.
-This fixture, prevents requests being made to the actual Strava API and instead
-registers responses that are based on examples from the published Strava API
-documentation. Example usages of this fixture can be found in the
-:mod:`stravalib.tests.integration.test_client` module.
-```
-
-We have setup the test suite to run on the stravalib package as installed.
-Thus when running your tests it is critical that you have a stravalib
+We have set up the test suite to run on the stravalib package as installed.
+Thus, when running your tests, it is critical that you have a stravalib
 development environment setup and activated with the stravalib package
 installed from your fork using pip `pip install .`
 
 You can run the tests using make as specified below. Note that when you run
 the tests this way, they will run in a temporary environment to ensure that
 they are running against the installed version of the package that you are working on.
 
-To run the test suite across all python versions that we support use:
+To run the test suite across all Python versions that we support use:
 
 ```
 nox -s tests
 ```
 
 `nox -s tests` does a few things:
 
-1. It create a temporary directory called `tmp-test-dir-stravalib` in which your tests are run. We create this test directory to ensure that tests are being run against the installed version of stravalib (with the most recent local development changes as installed) rather than the flat files located in the GitHub repository.
+1. It creates a temporary directory called `tmp-test-dir-stravalib` in which your tests are run. We create this test directory to ensure that tests are being run against the installed version of stravalib (with the most recent local development changes as installed) rather than the flat files located in the GitHub repository.
 2. It runs the tests and provides output (see below)
 3. Finally it removes the temporary directory
 
-To run tests for a specific python version use nox -s tests-python-version-here.
-
-For example, below we run tests for only Python 3.10.
-
-```bash
-nox -s tests-3.10
-```
-
-### Functional end-to-end test suite
```

Comment:
We dropped the functional test suite a while back because it required a token and app, etc. so I removed this section.